src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-function/GeneFunction.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { RootState } from 'src/store';
+import { GeneFunctionTabName } from 'src/content/app/entity-viewer/state/gene-view/entityViewerGeneViewState.ts';
+import { isEntityViewerSidebarOpen } from 'src/content/app/entity-viewer/state/sidebar/entityViewerSidebarSelectors';
+import { getEntityViewerActiveGeneFunction } from 'src/content/app/entity-viewer/state/gene-view/entityViewerGeneViewSelectors';
+import { setActiveGeneFunctionTab } from 'src/content/app/entity-viewer/state/gene-view/entityViewerGeneViewActions';
+
+import Tabs, { Tab } from 'src/shared/components/tabs/Tabs';
+import Panel from 'src/shared/components/panel/Panel';
+
+import styles from './GeneFunction.scss';
+
+const tabsData: Tab[] = [
+  { title: 'Proteins' },
+  { title: 'Variants' },
+  { title: 'Phenotypes' },
+  { title: 'Gene expression' },
+  { title: 'Gene ontology', isDisabled: true },
+  { title: 'Gene pathways' }
+];
+
+const tabClassNames = {
+  selected: styles.selectedTabName
+};
+
+type Props = {
+  isSidebarOpen: boolean;
```

Comment:
Here and in GeneRelstionship, I think that these components do not need to know about the existence of such a thing as sidebar somewhere in the other part of the interface. What they need to know is only whether their shape is"wide" or "narrow". Perhaps better call this prop `isNarrow`?

Reply:
Good point. I've renamed it to `isNarrow`.